app/forms/steps/conviction/conviction_schedule18_form.rb

```diff
@@ -4,6 +4,12 @@ class ConvictionSchedule18Form < BaseForm
       include SingleQuestionForm
 
       yes_no_attribute :conviction_schedule18
+
+      # As we reuse this form object in multiple views, this is the attribute
+      # that will be used to choose the locales for legends and hints.
+      def i18n_attribute
```

Comment:
Good to use the existing conventions.

As the text on the page doesn't vary depending on the subtype, maybe it would make sense to use a static value here, and then only one translation value is needed?

Also, a test should probably be added for this method.